SOURCES/xcp-networkd.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=XCP networking daemon
+Documentation=man:xcp-networkd(1)
+After=network-online.target forkexecd.service message-switch.service syslog.target
```

Comment:
Please remove `network-online.target` here and below (and anywhere else you see it!).